cmd/ebrelayer/main.go

```diff
@@ -67,27 +70,43 @@ var rootCmd = &cobra.Command{
 	SilenceUsage: true,
 }
 
-//	initRelayerCmd : Initializes a relayer service run by individual
-//		validators which streams live events from a smart contract.
+//	initEthereumRelayerCmd : Initializes a relayer service run by individual
+//		validators which streams live events from an Ethereum smart contract.
 //		The service automatically signs messages containing the event
 //		data and relays them to tendermint for handling by the
 //		EthBridge module.
 //
-func initRelayerCmd() *cobra.Command {
-	initRelayerCmd := &cobra.Command{
-		Use:   "init [web3Provider] [contractAddress] [eventSignature] [validatorFromName] --chain-id [chain-id]",
-		Short: "Initializes a web socket which streams live events from a smart contract",
+func initEthereumRelayerCmd() *cobra.Command {
+	initEthereumRelayerCmd := &cobra.Command{
+		Use:   "initEth [web3Provider] [contractAddress] [eventSignature] [validatorFromName] --chain-id [chain-id]",
+		Short: "Initializes a web socket which streams live events from a smart contract and relays them to the Cosmos network",
 		Args:  cobra.ExactArgs(4),
 		// NOTE: Preface both parentheses in the event signature with a '\'
-		Example: "ebrelayer init wss://ropsten.infura.io/ws ec6df30846baab06fce9b1721608853193913c19 LogLock(bytes32,address,bytes,address,uint256,uint256) validator --chain-id=testing",
-		RunE:    RunRelayerCmd,
+		Example: "ebrelayer initEth wss://ropsten.infura.io/ws 05d9758cb6b9d9761ecb8b2b48be7873efae15c0 LogLock(bytes32,address,bytes,address,string,uint256,uint256) validator --chain-id=testing",
+		RunE:    RunEthereumRelayerCmd,
+	}
+
+	return initEthereumRelayerCmd
+}
+
+//	initCosmosRelayerCmd : Initializes a Cosmos relayer service run by individual
+//		validators which streams live events from the Cosmos network and then
+//		relaying them to an Ethereum smart contract
+//
+func initCosmosRelayerCmd() *cobra.Command {
+	initCosmosRelayerCmd := &cobra.Command{
+		Use:     "initCos [tendermintProvider] [web3Provider] [contractAddress] [privateKey]",
```

Comment:
```suggestion
		Use:     "cosmos [tendermintNode] [web3Provider] [contractAddress] [privateKey]",
```

Side question, are there any security implications with passing the private key directly?

Reply:
Not directly since the cmd is run locally, but I think using an environment variable for the private key would be cleaner here.

Reply:
https://github.com/cosmos/peggy/pull/77/commits/6f01e466c0910fb7574bfa0f8fe99cf47c944abb